### PR TITLE
Config updation

### PR DESCRIPTION
**Pull request by vishalnagda1**
- update the bindings of ncmcpp to seek music back.
- add alias of python to point python3 for default terminal
- update the twitch configs

### CODE REVIEW

File: .config/shell/profile [hunk 1]
```diff
@@ -173,3 +173,7 @@ export VALE_CONFIG_PATH="${HOME}/.config/vale/vale.ini"
 # Loads support for direnv to load .envrc defined in specific directory
 command -v direnv > /dev/null && eval "$(direnv hook zsh)"
 
+eval "$(pyenv init --path)"
```
Comment (guruor) [inline .config/shell/profile:176]: Initialising pyenv was causing performance issue with shell loading.
So we are using https://github.com/davidparsson/zsh-pyenv-lazy plugin in zshrc.

File: .config/shell/profile [hunk 2]
```diff
@@ -173,3 +173,7 @@ export VALE_CONFIG_PATH="${HOME}/.config/vale/vale.ini"
...
+
+alias python=python3
```
Comment (guruor) [inline .config/shell/profile:178]: Better keep it in aliasrc-private.

File: .config/shell/profile (hunk 1 above)
Comment (vishalnagda1) [reply .config/shell/profile:176]: Got your point.

File: .config/shell/profile (hunk 2 above)
Comment (vishalnagda1) [reply .config/shell/profile:178]: Got your point, will update in aliasrc file.

**Comment by vishalnagda1** [comment]
The changes are no more needed so closing the PR.

**Pull request closed without merging**